.github/workflows/pages.yml

```diff
@@ -15,46 +15,41 @@ jobs:
   build:
     runs-on: ubuntu-20.04
     container:
-      image: ubuntu:23.10
+      image: fedora:latest
     steps:
       - name: Install dependencies
-        env:
-          DEBIAN_FRONTEND: noninteractive
         run: |
-          apt-get update
-          apt-get upgrade -y
-          apt-get install -y --no-install-recommends \
+          dnf install -y \
             desktop-file-utils \
             fuse3 \
             gcc \
             gettext \
             gnome-desktop-testing \
-            gtk-doc-tools \
-            libcap2-bin \
-            libflatpak-dev \
-            libfontconfig1-dev \
-            libfuse3-dev \
-            libgdk-pixbuf-2.0-dev \
-            libgeoclue-2-dev \
-            libglib2.0-dev \
-            libjson-glib-dev \
-            libpipewire-0.3-dev \
-            libportal-dev \
-            libsystemd-dev \
+            gtk-doc \
+            libcap \
+            libcap-devel \
+            flatpak-devel \
+            fontconfig-devel \
+            fuse3-devel \
+            gdk-pixbuf2-devel \
+            geoclue2-devel \
+            glib2-devel \
+            json-glib-devel \
+            pipewire-devel \
+            libportal-devel \
+            systemd-devel \
             llvm \
             meson \
-            python3-gi \
+            python3-gobject \
```

Comment:
Please sort this list alphabetically (don't ignore the `lib`  prefix :slightly_smiling_face:)

Reply:
Didn't even cross my mind, that it was alphabetical :)